java.html.markdown

```diff
@@ -99,7 +99,21 @@ public class LearnJava {
         System.out.println(fooString);
         System.out.println(barString);
         System.out.println(bazString);
-
+        
+        // String Building
+        // #1
+        String plusConcatenated = "Strings can " + "be concatenated " + "via + operator";
+
+        // #2
+        StringBuilder builderConcatenated = new StringBuilder();
```

Comment:
`StringBuilder` is rather rarely used. I think I run into a good use case once every three years. We could include it if you feel strongly, but at least also include an explanation of its advantage over using just `+`. My vote is to take this out though.